repository/src/test/java/uk/gov/hmcts/ccd/definition/store/repository/JsonUtilsTest.java

```diff
@@ -30,17 +30,15 @@ public void shouldGetObject_whenFromStringToLocalDate() {
         assertThat(JsonUtils.fromString(s, LocalDate.class), is(LocalDate.of(2020, 02, 20)));
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void shouldFail_whenObjectCannotBeMappedAsString() {
         final LocalDate date = LocalDate.of(2020, 02, 20);
         final String s = JsonUtils.toString(date);
 
-        try {
+        IllegalArgumentException ex = assertThrows(IllegalArgumentException.class, () -> {
             JsonUtils.fromString(s, String.class);
-        } catch (IllegalArgumentException ex) {
-            Assertions.assertThat(ex).hasMessageContaining(" cannot be transformed to Json object");
-            throw ex;
-        }
+        });
+        Assertions.assertThat(ex).hasMessageContaining(" cannot be transformed to Json object");
```

Comment:
Two different versions of `assertThat()` are used in this class (`org.hamcrest.MatcherAssert.assertThat` and `org.assertj.core.api.Assertions.assertThat`). Can this be simplified to use just one? The assertj one is the least frequently used one.